config/checks/check-eof-newline.bat

```diff
@@ -0,0 +1,30 @@
+@echo off
+rem Checks that all text files end with a newline.
+
+set temp=check-eof-newline~%RANDOM%.tmp
```

Comment:
is there a chance where the fact that RANDOM generates a random integer from 0 to 32767 only, causes issues? e.g., if the batch files are run concurrently/files are overwritten/reading from wrong file due to the same random integer?

Reply:
Currently, the files are run sequentially, so there should be no issue. The random variable at the back is just for extra safety. Each script outputs its output into a temp file with a different name (e.g. `check-eof-newline~%RANDOM%.tmp` and `check-line-endings~%RANDOM%.tmp`, so if in the future, even if its refactored such that the three checks run concurrently, there will be no collision as they all have different temp file names.